app/controllers/basespace_controller.rb

```diff
@@ -88,51 +69,15 @@ def samples_for_project
       return
     end
 
-    # 1024 is the maximum limit allowed by Basespace.
-    # If users reach this limit, we will need to implement multiple requests to fetch all the samples.
-    response = HttpHelper.get_json(
-      BASESPACE_PROJECT_DATASETS_URL % basespace_project_id,
-      { limit: BASESPACE_PAGE_SIZE },
-      "Authorization" => "Bearer #{access_token}"
-    )
+    samples = samples_for_basespace_project(basespace_project_id, access_token)
```

Comment:
as above, rename to `bs_samples = samples_for_basespace_project(basespace_project_id, access_token)`